app/blueprints/search.py

```diff
@@ -1,10 +1,13 @@
 import logging
 
+from ailab.db import DBError
+from ailab_llama_search import search as llama_search
 from flask import Blueprint, current_app, jsonify, request
 from index_search import AzureIndexSearchError, search
 
-from app.ailab_db import DBError, ailab_db_search
+from app.ailab_db import ailab_db_search
 from app.blueprints.common import create_error_response
+from app.config import Config
 from app.finesse_data import FinesseDataFetchException, fetch_data
 from app.utils import sanitize
```

Comment:
Be careful with naming you module `utils` as it could create maintenance problem.

[Stop naming your python modules "utils"](https://breadcrumbscollector.tech/stop-naming-your-python-modules-utils/#:~:text=Do%20not%20use%20utils%20as%20a%20name%20for,utils%2C%20try%20grouping%20code%20by%20its%20theme%20-)

Reply:
Good point. Let me fix that along with other refactors I had in mind to reduce code complexity in this issue #53.